db/re/item_db_equip.yml

```
@@ -103014,7 +103012,7 @@ Body:
     EquipLevelMin: 99
     Refineable: true
     Script: |
-      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+3*(getrefine()/2);
+      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+2*(getrefine()/2);
```

It was fine, 3% per 2 refine (KRO description)
```suggestion
      bonus2 bSkillAtk,"RA_CLUSTERBOMB",5+3*(getrefine()/2);
```

---

Thanks